specification/attributes/null_handling.md

```diff
@@ -0,0 +1,33 @@
+# Null Handling
+
+Cost data rows that don’t have a value that can be presented for a column must be handled in a consistent way to reduce
```

Comment:
Do we have a stance on smart/curly quotes yet?
```suggestion
Cost data rows that don't have a value that can be presented for a column must be handled in a consistent way to reduce
```

Reply:
Did some digging on this yesterday. I'm not a 100% sure. From my research, they are generally considered a positive if its using smart quotes / considered more readable. Doesn't seem like the linter has support to fail based on one or the other (i'm sure we could create a rule if we wanted to). Tempted to say since most editors automatically does the conversion, we should stick to the 'smart' kind but very much open to ideas/suggestions. We can also go clean up later in bulk if needed.